app/controllers/bgeigie_imports/not_processed_controller.rb

```diff
@@ -4,7 +4,7 @@ class NotProcessedController < ApplicationController
     before_filter :require_moderator
 
     def index
-      @bgeigie_imports = BgeigieImport.submitted.unapproved.page(params[:page])
+      @bgeigie_imports = BgeigieImport.where(status: :unprocessed).unapproved.page(params[:page])
```

Comment:
A method helper like https://github.com/Safecast/safecastapi/blob/master/app/models/bgeigie_import.rb#L23 could be nice here.

Reply:
this change was not for this PR, so reverted